SpatialOS/scripts/download_dependencies.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+cd "$(dirname "$0")"
+source ./utils.sh
+
+TOOLS_DIR="$(pwd)/../tools/${SDK_VERSION}"
+mkdir -p "${TOOLS_DIR}"
+
+retrievePackage() {
+  TYPE=$1
+  PACKAGE=$2
+  TARGETDIR=$3
+
+  pushd "${TOOLS_DIR}"
+  if [ ! -d "${TARGETDIR}" ]; then
+    spatial package get --force --unzip "${TYPE}" "${PACKAGE}" "${SDK_VERSION}" "${TARGETDIR}"
+  fi
+  popd
+}
+
+# Get the tools:
+# * Spatial Schema compiler
+# * Standard Library Schemas
+# * C# SDK
+# * Core SDK for all platforms to enable building workers for MacOS, Windows or Linux
+retrievePackage "tools" "schema_compiler-x86_64-${PLATFORM_NAME}" "schema_compiler"
+retrievePackage "schema" "standard_library" "standard_library"
```

Comment:
Does that mean the standard library is unzipped into the tools dir? Is it even copied into the worker dirs?

Reply:
Ok, after looking into this more, I think this is an oversight on my part. We basically do not know where to put the unzipped std lib schema files. I will update the design doc. Sorry about this.

Reply:
Ok, I've now made the following change to the design doc: 
- The built out schema descriptor will bin in schema/bin
- The unzipped std lib schema files will be in schema/improbable
- The project specific schema will be in schema/<project-name, FlexibleProjectExample in this case>.

Again, sorry

Reply:
thanks! done